### ISSUE TITLE
Handle omnipods with set-bonus for different number of pieces

### ISSUE THREAD

**Issue by LiSongMWO** [lisongmechlab/lsml#824]
Example
```
  <Set name="vpr-dilgd">
    <SetBonuses>
      <Bonus PieceCount="8">
        <Quirk name="xpbonus_multiplier" value="0.025" />
        <Quirk name="ammocapacity_csrm_additive" value="300" />
        <Quirk name="missile_range_multiplier" value="0.10" />
        <Quirk name="energy_cooldown_multiplier" value="-0.7" />
      </Bonus>
      <Bonus PieceCount="6">
        <Quirk name="missile_spread_multiplier" value="-0.20" />
        <Quirk name="missile_range_multiplier" value="0.2" />
		<Quirk name="missile_velocity_multiplier" value="0.2" />
      </Bonus>
    </SetBonuses>
```

**Comment by LiSongMWO** [comment]
It seems like the set bonuses are additive. So if you have 8 pieces, you get the quirks for the 8 piece bonus AND the 6 piece bonus.

**Comment by LiSongMWO** [comment]
Some work done in commit f10d7c5e.

**Comment by LiSongMWO** [comment]
This is now fixed, however issue #826 is now a thing. Not to be fixed in 2.1.1 tho.
